stock_orderpoint_generator/models/orderpoint_template.py

```diff
@@ -184,8 +184,9 @@ def create_orderpoints(self, products):
         """Create orderpoint for *products* based on these templates.
         :type products: recordset of products
         """
-        self._disable_old_instances(products)
```

Comment:
@sergiocorato Why this change ? Is there a problem using the recordset ?

Reply:
I fixed it, thanks.